pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -242,10 +242,50 @@ func StartVirtlog(stopChan chan struct{}) {
 
 			err := cmd.Start()
 			if err != nil {
-				log.Log.Reason(err).Error("failed to start libvirtd")
+				log.Log.Reason(err).Error("failed to start virtlogd")
 				panic(err)
 			}
 
+			go func() {
```

Comment:
Can we add some unit tests for this to `libvirt_helper_test.go` which transforms typical qemu logs to kubevirt logs?

Reply:
Sure, I'll be adding that shortly. Thanks!